### DISCUSSION TITLE
求更新arm8架构的安装包

### DISCUSSION THREAD

**Discussion by sss-link** [messense/aliyundrive-webdav#420] [Q&A] [answered]
硬路由ax86u

**Comment by messense** [comment, marked as answer]
armv8 即 aarch64.

**Reply by youe360** [reply]
【2022年08月11日 16:39:19】: 运行安装脚本...
【2022年08月11日 16:39:19】: ========================== step 2 ===============================
【2022年08月11日 16:39:19】: 本插件适用于【koolshare merlin aarch64 / merlin_hnd】固件平台，你的平台不能安装！！！
【2022年08月11日 16:39:19】: ========================== step 3 ===============================
【2022年08月11日 16:39:19】: ks_tar_install.sh: 因为插件aliyundrivewebdav安装失败！退出离线安装！
【2022年08月11日 16:39:19】: 删除文件夹-1：/tmp/aliyundrivewebdav
【2022年08月11日 16:39:19】: 删除文件：/tmp/aliyundrivewebdav-merlin-arm384-aarch64.tar.gz
【2022年08月11日 16:39:19】: ============================ end ================================
ax86U  aarch64 无法安装

**Reply by xllfe** [reply]
aarch64 有好几个啊

**Reply by youe360** [reply]
老大，没法安装，能否帮忙解决下，愿意付费

**Reply by messense** [reply]
@youe360 https://github.com/messense/aliyundrive-webdav/discussions/442

**Reply by wellonhhh** [reply]
能安装的

**Comment by wellonhhh** [comment]
可以安装
